sql/statistics.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Introduction to Statistics
+category: user guide
+---
+
+# Introduction to Statistics
+
+Based on the statistics, the TiDB optimizer chooses the most efficient query execution plan. The statistics collect table-level and column-level information. The statistics of tables include the total number of rows and the number of modified rows. The statistics of columns include the number of different values, the number of `NULL`, and the histogram of one column.
+
+## Collect Statistics
+
+### Manual Collection
+
+You can run the `ANALYZE` statement to collect statistics.
+
+Syntax:
+
+```sql
+ANALYZE TABLE TableNameList
+> The statement collects statistics of all the tables in the `TableNameList` list. 
+
+ANALYZE TABLE TableName INDEX IndexNameList
+> The statement collects statistics of the index columns on all the `IndexNameList` lists in the `TableName` tables. 
```

Comment:
all the `IndexNameList` lists in the `TableName` tables ->
all `IndexNameList` in `TableName`.